### ISSUE TITLE
Spaces are trimmed when showing shell logs

### ISSUE THREAD

**Issue by vit9696** [buildbot/buildbot#7774]
Consider a `ShellCommand` that prints something like this:

```
[  1/ 10] Test1                      \x1b[32;1mOK\x1b[0m
[  2/ 10] Test2                      \x1b[32;1mOK\x1b[0m
[  3/ 10] Test3                      \x1b[32;1mOK\x1b[0m
…
[ 10/ 10] Test10                     \x1b[32;1mOK\x1b[0m
```

New React UI will print:

```
[ 1/ 10] Test1 OK
[ 2/ 10] Test2 OK
[ 3/ 10] Test2 OK
…
[ 10/ 10] Test10 OK
```

While the expected output is:

```
[  1/ 10] Test1                       OK
[  2/ 10] Test2                       OK
[  3/ 10] Test3                       OK
…
[ 10/ 10] Test10                      OK
```

The issue is caused by using `span` in formatting without defining `white-space` policy. If one adds `white-space: pre;` it works correctly again.

**Comment by mokibit** [comment]
@p12tic, 
issue was fixed in https://github.com/buildbot/buildbot/pull/7816.

**Comment by vit9696** [comment]
This one is indeed fixed in 4.0.2. Thank you.
